blueprints/app/files/package.json

```diff
@@ -24,6 +24,8 @@
     "test:ember": "ember test"
   },
   "devDependencies": {
+    "@babel/eslint-parser": "^7.15.8",
```

Comment:
**bug (blocking)**

Before, `babel-eslint` was added to `package.json` via the `else` clause. You'll want to add `@babel/eslint-parser` and `@babel/plugin-proposal-decorators` conditionally.

Pseudocode:

```jsons
<% if (!typescript) { %>
"@babel/eslint-parser": "...",
"@babel/plugin-proposal-decorators": "...",
<% } %>
```